content/root/best-practice-guide.textile

```diff
@@ -91,6 +91,16 @@ The "@attach@":/realtime/channels#attach method attaches a client to the channel
 
 The "@subscribe@":/realtime/channels#subscribe method registers a subscribe listener for messages received on the channel. @subscribe@ is a client-side operation, meaning Ably is unaware of whether or not a client is subscribed to the channel. @attach@ is called implicitly by the @subscribe@ method.
 
+h3(#detach-vs-unsubscribe). Detaching versus unsubscribing from a channel
+
+It is important to understand the difference between detaching and unsubscribing from a channel, and that messages will continue to be sent to clients if they only call the @unsubscribe()@ method.
+
+The "@detach()@":/realtime/channels#detach method detaches a client from the channel. The client will no longer receive any messages published to the channel.
+
+The "@unsubscribe()@":/realtime/channels#unsubscribe method removes message listeners for the channel. @unsubscribe()@ is a client-side operation, meaning Ably is unaware of whether or not a client has unsubscribed from the channel. Messages will continue to be streamed to the client until @detach()@ is called.
+
+Calling @subscribe()@ for a channel "implicitly attaches":/realtime/channels#implicit-attach the client to the channel as well. It is important to note that if you call @subscribe()@ followed by @unsubscribe()@, the client remains attached to the channel.
```

Comment:
> Calling @subscribe()@ for a channel

Was `subscribe` meant to be a hyperlink, as done for detach and unsubscribe?

Reply:
Good point! Added that in.